build_tools/benchmarks/suites/iree_benchmarks.py

```diff
@@ -0,0 +1,81 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Defines IREE benchmarks."""
+
+from itertools import product
+from typing import List, Tuple
+from .device_specs.linux_x86_64 import GCP_C2_STANDARD_16_DEVICE_SPEC
+from .models.model_group import MOBILE_MODEL_GROUP
+from .definitions.common import DeviceArchitecture, DevicePlatform, RANDOM_MODEL_INPUT_DATA
+from .definitions.iree import (IreeBenchmarkCompileSpec, IreeBenchmarkRunSpec,
```

Comment:
So similar to the above comment on package imports, rather than having every class prefixed with "Iree" and importing them directly, you could import the whole iree package and then have these as qualified names like `iree.BenchmarkCompileSpec`. Similarly, you could have `model_group.MOBILE`

Reply:
Done.

Since we use relative import, it's not possible to have `from . import definitions.iree` and use the full qualified `definitions.iree` in the code. I renamed some modules to make their names more verbose.

Reply:
Can we have `from .definitions import iree`?

Reply:
I feel like `import iree` is obscure? Especially since we have `import iree.compiler`

Reply:
Good point. You're right. Would indeed be nice if we could have it be `definitions.iree.FooSpec`. I don't really understand the rules of Python imports though

Reply:
I believe after we have #10065, the code can be refactored into a better form. But I'm still not sure about that hacky solution.